mbs_results/pivot_imputation_value.py

```diff
@@ -0,0 +1,169 @@
+import pandas as pd
+
+
+def merge_counts(
+    input_df: pd.DataFrame,
+    count_df: pd.DataFrame,
+    input_cell: str,
+    count_cell: str,
+    input_date: str,
+    count_date: str,
+    identifier: str,
+) -> pd.DataFrame:
+    """
+    Returns input data with f_count and b_count merged on.
+
+    Parameters
+    ----------
+    input_df : pd.DataFrame
+        Reference dataframe with identifier, date, sic, cell, forward, backward,
+        construction, question, imputed_value
+    count_df : pd.DataFrame
+        DataFrame with group, period, f_count and b_count
+    input_cell : str
+        name of column in input_df dataframe containing cell variable
+    count_cell : str
+        name of column in count_df dataframe containing cell variable
+    input_date : str
+        name of column in input_df dataframe containing date variable
+    count_date : str
+        name of column in count_df dataframe containing date variable
+    identifier : str
+        name of column in input_df containing identifier variable
+
+    Returns
+    -------
+    Dataframe resulting from the left-join of input_df and count_df on the cell and
+    date columns.
+    """
+    df_merge = pd.merge(
+        input_df,
+        count_df,
+        how="left",
+        left_on=[input_cell, input_date],
+        right_on=[count_cell, count_date],
+    ).astype({identifier: "int"})
+
+    return df_merge.drop(columns=[count_cell, count_date])
+
+
+def pivot_imputation_value(
+    df: pd.DataFrame,
+    identifier: str,
```

Comment:
is identifier arg needed ?

Reply:
This has been removed; also removed references to 'identifier' in the test file.